### PR TITLE
simple command line client

### PR DESCRIPTION
**Pull request by vii**
This simple command line client makes it easy to activate switches and view status for all the supported command classes in zwave-ilb-dotnet

For example, to turn off the binary switch at the third address

ZWaveCommandLine.exe 'SwitchBinary.Set(node3, 0)' 

Would love to know a better way to integrate this into your library!

### CODE REVIEW

**Comment by genemars** [comment]
This should be moved outside perhaps merged into the Test.ZWave project.

File: ZWaveLib/ZWaveController.cs [hunk 1]
```diff
@@ -182,7 +182,17 @@ public void Dispose()
         public void Connect()
         {
             LoadNodesConfig();
-            new Thread(() => { serialPort.Connect(); }).Start();
+
+            new Thread(() =>
+            { 
+                if (!serialPort.Connect())
+                {
+                    Utility.logger.Error("Failed to connect to serial port {0}", serialPort);
```
Comment (vii) [inline ZWaveLib/ZWaveController.cs:190]: this just causes nicer error messages when the serial port is misspecified

File: Command.ZWave/Command.ZWave.csproj [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <Externalconsole>true</Externalconsole>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="System" />
+    <Reference Include="NLog">
+      <HintPath>..\packages\NLog.4.1.0\lib\net45\NLog.dll</HintPath>
+    </Reference>
+    <Reference Include="Mono.CSharp" />
```
Comment (vii) [inline Command.ZWave/Command.ZWave.csproj:36]: this tool has a dependency on Mono so I thought it best to make it a separate package, so the Test.ZWave tool would not be affected

**Comment by vii** [comment]
sorry for responding so slowly - I hope this is better?

**Comment by genemars** [comment]
I think it's best to create this as a separate project an utility tool that shows how to use ZWaveLib.
I will be glad to link it in the README.md of this repo.

**Pull request closed without merging**